ai-ml/jark-stack/terraform/addons.tf

```diff
@@ -319,6 +354,50 @@ module "data_addons" {
   ]
 }
 
+#---------------------------------------------------------------
+# S3 CSI Mountpoint Driver 
+#---------------------------------------------------------------
+
+resource "helm_release" "aws_mountpoint-s3-csi-driver" {
```

Comment:
Could we not use the EKS Managed add-on for this deployment?

Reply:
I checked again and found tolerations of all taints is supported from v1.8.0 or later. Let me update this.